feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.component.html

```diff
@@ -0,0 +1,49 @@
+<ng-container *ngIf="orderEntry$ | async as orderEntry">
+  <ng-container *ngIf="isBundleBasedConfigurator(orderEntry)">
+    <ng-container *ngIf="(numberOfLineItems$ | async) > 0">
+      <div class="cx-number-items">
+        {{
+          'configurator.header.items'
+            | cxTranslate: { count: numberOfLineItems$ | async }
+        }}
+      </div>
+
+      <a role="link" (click)="toggleItems()" tabindex="0">
+        <div class="cx-toggle-hide-items">
+          <ng-container *ngIf="hideItems">
+            {{ 'configurator.header.show' | cxTranslate }}
+          </ng-container>
+          <ng-container *ngIf="!hideItems">
+            {{ 'configurator.header.hide' | cxTranslate }}
+          </ng-container>
+        </div>
+      </a>
+
+      <div class="cx-item-infos" [class.open]="!hideItems">
+        <div
+          class="cx-item-info"
+          *ngFor="let lineItem of retrieveLineItems(orderEntry)"
```

Comment:
Each time clicking `hide` or `show`, function `retrieveLineItems(orderEntry)` is called. So, try to not use function calls in template.

Reply:
Pick!

Reply:
It was a good point. I took it into account!